recipe/conda_build_config.yaml

```diff
@@ -13,5 +13,6 @@ channel_targets:
   - conda-forge main
   - conda-forge debug
 zip_keys:
-  - - build_type
+  - 
+    - build_type
```

Comment:
BTW: No typo -- either is fine. But you can leave it as is now ;).

Reply:
Oh I see, sorry I referred to the [docs](https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#coupling-keys) and that is what it suggested. I'm still curious to know why we use a nested list in `zip_keys` when the docs suggest either is fine?